app/main.py

```diff
@@ -1,5 +1,73 @@
+from __future__ import annotations
+
+
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot(Cargo):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
+    ) -> None:
+        super().__init__(weight)
+        self.name = name
+        self.coords = [0, 0] if coords is None else coords
+
```

Comment:
```suggestion
        self.coords = coords or [0,0]

```
your solution is cool. but look at this